source/overloading.tex

```diff
@@ -4199,7 +4199,7 @@
 \pnum
 A declaration whose \grammarterm{declarator-id} is a
 \grammarterm{literal-operator-id} shall be a declaration of a namespace-scope
-function or function template (it could be a friend
+function or function template (it can be a friend
 function\iref{class.friend}), an explicit instantiation or specialization of a
 function template, or a \grammarterm{using-declaration}\iref{namespace.udecl}.
```

Comment:
"can" makes it sound like this is an exhaustive list, as if "it can be ..., and nothing else". This seems a little better:

```suggestion
function or function template (for example, it can be a friend
function\iref{class.friend}), an explicit instantiation or specialization of a
function template, or a \grammarterm{using-declaration}\iref{namespace.udecl}.
```

Reply:
fixed